.github/workflows/deployment.yml

```diff
@@ -0,0 +1,28 @@
+name: Deployment
+
+on:
+  # Uncomment once we are ready to deploy 
+  #push:
```

Comment:
Will the website go down briefly, while the container rebuilds, each time there is a push to main?

Reply:
Yeah that's right, the website will go down as it's rebuilds, but the DB is persisted so it should be okay (we won't lose any of the data and the UI will start up back up where it left off). 

I think we can go with this for now, the downtime shouldn't be too bad. If we really wanted to ensure the website never goes down there are some deployment strategies such as [blue/green and red/black](https://octopus.com/blog/blue-green-red-black ), but imo it might be overkill. 

Might be worth considering in the future though!

Reply:
Sounds good, yeah I don't see it being much of an issue, good to know there's some solutions if it does become a problem. I sure hope the website pops off! 😆